cmake/Buildfind-superquadric.cmake

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2022  iCub Tech Facility, Istituto Italiano di Tecnologia
+# Authors: Ugo Pattacini <ugo.pattacini@iit.it>
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(YARP QUIET)
+find_or_build_package(ICUB QUIET)
+find_or_build_package(ICUBcontrib QUIET)
+
+ycm_ep_helper(find-superquadric TYPE GIT
+                                STYLE GITHUB
+                                REPOSITORY robotology/find-superquadric.git
+                                DEPENDS YARP
+                                        ICUB
+                                        ICUBcontrib
```

Comment:
@pattacini can I remove the dependency on `ICUBcontrib`? At a first glance, the repo does not depend on it.